packages/guides/src/Handlers/RenderCommand.php

```diff
@@ -10,10 +10,10 @@
 
 final class RenderCommand
 {
-    /** @param DocumentNode[] $documents */
+    /** @param iterable<DocumentNode> $documents */
```

Comment:
`iterate` returns a generator, but looking at the rest of a project, the fact that `getDocuments()` is an array is never used.

Reply:
I think it should be fine then